templates/probot/files/README.md

```diff
@@ -0,0 +1,91 @@
+<a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/platformsh/template-builder/master/templates/probot/.platform.template.yaml&utm_content=probot&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
+    <img src="https://platform.sh/images/deploy/lg-blue.svg" alt="Deploy on Platform.sh" width="180px" />
+</a>
+</p>
+
+This template builds a simple GitHub App using [Probot](https://github.com/probot/probot).
+
+Probot is a framework for building GitHub Apps in Node.js.
+
+## Services
+
+* Node.js 12
+
+## Post-install
+
+1. **Register the App on GitHub**
+
+   When the app has deployed, you will need to first register the app with GitHub by clicking the **Register GitHub App** button on the homepage. Name the application anything you like, then click the **Create GitHub App** button. This will automatically save your `WEBHOOK_SECRET`, `PRIVATE_KEY`, and `APP_ID` variables to a `.env` file in your Platform.sh environment.
+
+2. **Clone repository locally**
+
+   Return to the management console for the project. Enter the master environment and copy the clone command under the `GIT` dropdown  button.
+
+3. **Set up for production: Adding an API token**
+
+   A great thing about GitHub Apps on Platform.sh is that you are not only able to setup a production application, but that you can also have `staging` and `development` versions of your application, themselves just as production-ready as your `master` branch, that you can test on GitHub.
+
+   All of the credentials we have received from GitHub at this point sit in directories with write access at the moment, and they would be overwritten once we start branching and merging our apps.
+
+   Luckily, it\'s very easy to install the Platform.sh into your application container, and set environment variables specific to each branch that will not be overwritten. All of the code to install the CLI is already in this demo, all you need to do is add an API token to the project and Platform.sh will take care of the rest.
+
+   Follow the [instructions in our public documentation](https://docs.platform.sh/development/cli/api-tokens.html#obtaining-a-token) to create an API token.
+
+   Once you have that token, copy it into the command below.
+
+   ```
+   platform variable:create --level project --name env:PLATFORMSH_CLI_TOKEN --sensitive true --value YOUR_API_TOKEN
+   ```
+
+   You will only need to add your API token once to a project. After it has been added, every branch will be able to add environment variables automatically with no further changes necessary.
+
+4. **Set up for production: setting `NODE_ENV`**
+
+   Inside the local repository, find the `.environment` file. You will see that `NODE_ENV` is currently set for `development`.
+
+   Update that file to read `NODE_ENV=\"production\"`. Once you have done that, commit and push the change.
+
+   ```
+   git commit -am \"Set for production\" && git push origin master
```

Comment:
Are the backslashes here necessary?  It seems like that's excessive escaping.  And in the line above.